docs/book/GLOSSARY.md

```diff
@@ -37,10 +37,18 @@ Services beyond the fundamental components of Kubernetes.
 * __Core Add-ons__: Addons that are required to deploy a Kubernetes-conformant cluster: DNS, kube-proxy, CNI.
 * __Additional Add-ons__: Addons that are not required for a Kubernetes-conformant cluster (e.g. metrics/Heapster, Dashboard).
 
+# B
+
 ## Bootstrap
 
 The process of turning a server into a Kubernetes node. This may involve assembling data to provide when creating the server that backs the Machine, as well as runtime configuration of the software running on that server.
 
+## Bootstrap cluster
+
+A temporary cluster that is used to provision a Target cluster.
```

Comment:
Since @dlipovetsky brought up the Management/Workload cluster nomenclature, and to avoid causing confusion with those terms, maybe this should call out `Target Management cluster` rather than just `Target cluster`